resources/views/emails/reset-password-link.blade.php

```diff
@@ -0,0 +1,14 @@
+@component('mail::message')
+<h4>Reset password</h4>
+<p>Hi {{ $user->name }}!</p>
+<p>You are receiving this email because we received a password reset request for your account.</p>
+<p>If you did not request a password reset, no further action is required.</p>
+@component('mail::button', ['url' => action([\App\Http\Controllers\ForgotPasswordController::class, 'edit'], $token)])
+Reset password
+@endcomponent
+@component('mail::footer')
+<small>{{ config('app.name') }}</small>
+<br>
+<a href="{{ action([\App\Http\Controllers\SessionsController::class, 'create']) }}">tpuq.test</a>
```

Comment:
**NEVER** write in the code fixed website name. Use the config instead.
Always remember that your project later can be renamed to `vaspur` and you will need to modify ONLY env file.